Formula/s/sdl12-compat.rb

```diff
@@ -6,6 +6,11 @@ class Sdl12Compat < Formula
   license all_of: ["Zlib", "MIT-0"]
   head "https://github.com/libsdl-org/sdl12-compat.git", branch: "main"
 
+  livecheck do
+    url :stable
+    regex(/^release-(\d+(?:\.\d+)*)$/i)
```

Comment:
```suggestion
    regex(/^release[._-]v?(\d+(?:\.\d+)+)$/i)
```
 Just a couple tweaks to align with our typical regexes.